doc/source/configuration.rst

```diff
@@ -35,7 +35,10 @@ the theme can be one of the following:
 
```

Comment:
While you are at it, can you please document the key type parameter, too (the parameter that was not yet documented)?